Client/Frontend/Home/HistoryHighlights/FxHomeHistoryHighlightsVC.swift

```diff
@@ -0,0 +1,239 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+import Storage
+
+struct HistoryHighlightsCollectionCellConstants {
+    static let maxNumberOfItemsPerColumn = 3
+    static let maxNumberOfColumns = 3
+}
+
+class FxHomeHistoryHighlightsCollectionCell: UICollectionViewCell, ReusableCell {
+
+    // MARK: - Properties
+    var profile: Profile?
```

Comment:
I believe profile is not used in this class. And if it was, I suggest to pass it on the `init` of the viewModel. Makes more sense to me that the data is provided by the VM.